classes/options-general.php

```diff
@@ -777,7 +777,7 @@ class="radio-thumbnail<?php if ( 'SATELLITE' == $map_type ) {
 																	<img
 																		src="<?php echo SIMPLEMAP_URL; ?>/inc/images/map-type-satellite.jpg"
 																		width="100" height="100"
-																		style="border: 1px solid #999;"/><br/><?php _e( 'Satellite map', 'simplemap' ); ?>
+																		style="border: 1px solid #999;"/><br/><?php _e( 'Satellite map', 'SimpleMap' ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 154 characters

<!-- Comment by [SideCI](https://sideci.com) -->